cluster/addons/podsecuritypolicies/restricted.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted
+  annotations:
+    kubernetes.io/description: 'restricted allows minimal access for create pods to work.'
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+spec:
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - '*'
```

Comment:
This is not at all restrictive yet. I'd expect something more like https://github.com/kubernetes/kubernetes/blob/master/examples/podsecuritypolicy/rbac/policies.yaml#L18, possibly with some capability modifications as well. 

@jessfraz @pweil- for input here